sycl/source/detail/kernel_compiler/kernel_compiler_sycl.hpp

```diff
@@ -12,6 +12,8 @@
 #include <sycl/detail/export.hpp> // __SYCL_EXPORT
 #include <sycl/device.hpp>
 
+#include <detail/compiler.hpp> // sycl_device_binaries
```

Comment:
Should we use `<sycl/detail/compiler.hpp>` for consistency with imports above?

Reply:
`compiler.hpp` is in `source/detail` (not `include/detail`), so I can't include it as `sycl/detail/compiler.hpp`.